airbase/pom.xml

```diff
@@ -465,6 +465,11 @@
                         </argLine>
                     </configuration>
                     <dependencies>
+                        <dependency>
+                            <groupId>org.junit.jupiter</groupId>
+                            <artifactId>junit-jupiter-engine</artifactId>
+                            <version>5.8.0-M1</version>
```

Comment:
We should make the version a property and probably also include the dependencies so they are consistent
```xml
<dependency>
    <groupId>org.junit</groupId>
    <artifactId>junit-bom</artifactId>
    <type>pom</type>
    <version>${dep.junit.version}</version>
    <scope>import</scope>
</dependency>

Reply:
Should that go in the dependencyManagement section?

Reply:
Yes. I don't think that `dependencyManagement` affects plugin dependencies, so you probably still need the version property and the usage here